solidity/contracts/bridge/BridgeGovernanceParameters.sol

```diff
@@ -0,0 +1,1689 @@
+        self.redemptionTreasuryFeeDivisorChangeInitiated = block.timestamp;
+        emit RedemptionTreasuryFeeDivisorUpdateStarted(
+            _newRedemptionTreasuryFeeDivisor,
+            block.timestamp
+        );
+        /* solhint-enable not-rely-on-time */
+    }
+
+    /// @notice Finalizes the redemption treasury fee divisor amount update process.
+    /// @dev Can be called only by the contract owner, after the governance
+    ///      delay elapses.
+    function finalizeRedemptionTreasuryFeeDivisorUpdate(
+        RedemptionData storage self,
+        uint256 governanceDelay
+    )
+        external
+        onlyAfterGovernanceDelay(
+            self.redemptionTreasuryFeeDivisorChangeInitiated,
+            governanceDelay
+        )
+    {
+        emit RedemptionTreasuryFeeDivisorUpdated(
+            self.newRedemptionTreasuryFeeDivisor
+        );
+
+        self.newRedemptionTreasuryFeeDivisor = 0;
+        self.redemptionTreasuryFeeDivisorChangeInitiated = 0;
+    }
+
+    function getNewRedemptionTreasuryFeeDivisor(RedemptionData storage self)
+        internal
+        view
+        returns (uint64)
+    {
+        return self.newRedemptionTreasuryFeeDivisor;
+    }
+
+    /// @notice Begins the redemption tx max fee amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newRedemptionTxMaxFee New redemption tx max fee amount.
+    function beginRedemptionTxMaxFeeUpdate(
+        RedemptionData storage self,
+        uint64 _newRedemptionTxMaxFee
+    ) external {
+        /* solhint-disable not-rely-on-time */
+        self.newRedemptionTxMaxFee = _newRedemptionTxMaxFee;
+        self.redemptionTxMaxFeeChangeInitiated = block.timestamp;
+        emit RedemptionTxMaxFeeUpdateStarted(
+            _newRedemptionTxMaxFee,
+            block.timestamp
+        );
+        /* solhint-enable not-rely-on-time */
+    }
+
+    /// @notice Finalizes the redemption tx max fee amount update process.
+    /// @dev Can be called only by the contract owner, after the governance
+    ///      delay elapses.
+    function finalizeRedemptionTxMaxFeeUpdate(
+        RedemptionData storage self,
+        uint256 governanceDelay
+    )
+        external
+        onlyAfterGovernanceDelay(
+            self.redemptionTxMaxFeeChangeInitiated,
+            governanceDelay
+        )
+    {
+        emit RedemptionTxMaxFeeUpdated(self.newRedemptionTxMaxFee);
+
+        self.newRedemptionTxMaxFee = 0;
+        self.redemptionTxMaxFeeChangeInitiated = 0;
+    }
+
+    function getNewRedemptionTxMaxFee(RedemptionData storage self)
+        internal
+        view
+        returns (uint64)
+    {
+        return self.newRedemptionTxMaxFee;
+    }
+
+    /// @notice Begins the redemption timeout slashing amount amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newRedemptionTimeout New redemption timeout slashing amount amount.
```

Comment:
Begins the redemption timeout slashing amount amount update process. ->
Begins the redemption timeout amount update process.

@param _newRedemptionTimeout New redemption timeout slashing amount amount. ->
@param _newRedemptionTimeout New redemption timeout amount.

Reply:
[2056584](https://github.com/keep-network/tbtc-v2/pull/329/commits/205658478a76695bbe0f822cea8182268872cc05)